### PR TITLE
CAT-1697 Fix translation in CategoryBrickFactory

### PR DESCRIPTION
**Pull request by davidmarogy**
Fixed the Bug which appears when you don't use PocketCode in english, now we
check before we create all the Bricks in the Category Sound if there is a difference between the constant value for the Brick  and our translated Value. If there is a difference we use the value from the translation file.

### CODE REVIEW

**Comment by 84n4n4** [comment]
https://jenkins.catrob.at/view/All-Categories/view/Catroid/job/Catroid-CustomEmulator-Single-ClassAndPackage-Test/668/

**Comment by robertpainsi** [comment]
The commit message summery (header) is very general. Maybe you can improve it by mentioning the default value of the speak brick and the wrong language? So you get an idea about the commit without reading the Jira ticket nor the body.

Please add the detailed explanation about the pull request to the commit itself. I really like that you explained the effects of the commit for the speak brick in the brick factory. :smile_cat:

File: catroid/src/org/catrobat/catroid/ui/fragment/CategoryBricksFactory.java [hunk 1]
```diff
@@ -284,7 +284,11 @@
 		);
 		soundBrickList.add(new ChangeVolumeByNBrick(new Formula(defaultValueChangeVolumeBy)));
 
-		soundBrickList.add(new SpeakBrick(BrickValues.SPEAK));
+		if (context.getString(R.string.brick_speak_default_value) != BrickValues.SPEAK) {
+			soundBrickList.add(new SpeakBrick(context.getString(R.string.brick_speak_default_value)));
+		} else {
+			soundBrickList.add(new SpeakBrick(BrickValues.SPEAK));
+		}
```
Comment (robertpainsi) [inline catroid/src/org/catrobat/catroid/ui/fragment/CategoryBricksFactory.java]: Maybe it's better to remove the variable `BrickValues.SPEAK` entirely and always use `R.string.brick_speak_default_value`?

**Comment by robertpainsi** [comment]
Thank you, it's almost ready to merge :smile_cat:

Rewording the pull request header is fine too, of course. However, I was talking about the commit message :wink: Using the current pull request title as commit message will violate the header length of our [commit message guidelines](https://confluence.catrob.at/display/KNOWHOW/Commit+Message+Guidelines) (I guess you have access to the page). So you have to come up with another one :scream_cat:.

**Comment by thmq** [comment]
@davidmarogy, please do change the commit message, so I can finally merge this. (just make it shorter and more concise) :wink:

**Comment by avihric** [comment]
For sake of completeness: [Static checks](https://jenkins.catrob.at/job/Catroid-Build-SourceTests-StaticChecks/4215/)

**Comment by thmq** [comment]
@davidmarogy please remove unused imports etc.

**Comment by avihric** [comment]
I don't see any unused imports...

> For sake of completeness: Static checks

was to clarify that the source tests actually work, because the automated build had failures

**Pull request merged**